### DISCUSSION TITLE
1.26.0

### DISCUSSION THREAD

**Discussion by xaoxuu** [xaoxuu/hexo-theme-stellar#364] [Announcements]
**新特性**

- 侧边栏支持设置背景图、高速模糊、渐变效果
- 侧边栏主导航菜单支持设置图标、支持设置列数
- 支持在 `_data/icons.yml` 中配置图标
- 侧边栏新增 `linklist` 组件，支持图标、网格和列表布局
- 侧边栏 `markdown` 组件新增 `linklist` 参数，写法同 `linklist` 组件
- 大多数内置图标都可以通过在 `_data/icons.yml` 中覆盖同名配置来修改了
- 支持设置 article.type 来增大字体和图文间距，以优化图文类博客阅读体验

**重要变更**
- 移除 `seo_title` 字段，如果要隐藏页内标题，请设置 `h1: ''`
- 侧边栏主导航菜单由 `menu` 改为 `menubar` 
- 不再支持 `pin: true` 来置顶，因为官方已经支持该功能，方法为： `sticky: 数字`
- 搜索框不再是组件，而是独立功能，支持在 front-matter 和项目配置文件中覆盖参数
- wiki 配置文件中的 `path` 更名为 `base_dir` 

<hr /><em>This discussion was created from the release <a href='https://github.com/xaoxuu/hexo-theme-stellar/releases/tag/1.26.0'>1.26.0</a>.</em>

**Comment by chaofanx** [comment]
https://xaoxuu.com/  好像挂了

**Reply by chaofanx** [reply]
网站进不去了

**Reply by xaoxuu** [reply]
我正在弄服务器，先用xaoxuu.github.io

**Comment by cyliu-phy** [comment]
请问sidebar自定义背景颜色之后如何自动适配黑暗模式
<img width="453" alt="image" src="https://github.com/xaoxuu/hexo-theme-stellar/assets/71125015/fdfbbf72-ba47-4372-b9b7-6ba1480d0396">

**Reply by xaoxuu** [reply]
目前可以在颜色上设置透明度并开启blur来实现，不过后续打算增加一个配置使亮色和深色分开设置，这样更容易配色一些。

**Reply by xaoxuu** [reply]
你试一下这个配色方案：

```
linear-gradient(225deg,rgba(14,135,241,.4),rgba(93,118,162,.4),rgba(176,112,64,.4),rgba(201,63,54,.4))
```
